.gitlab-ci.yml

```diff
@@ -629,6 +643,8 @@ test_accep_vexpress_qemu_flash:
     JOB_BASE_NAME: mender_vexpress_qemu_flash
   after_script:
     - export WORKSPACE=$(realpath ${CI_PROJECT_DIR}/..)
+    - mkdir -p stage-artifacts
+    - cp $WORKSPACE/vexpress-qemu-flash/vexpress-qemu-flash_release_1_*.mender stage-artifacts/
```

Comment:
This is wrong, `vexpress-qemu-flash` does not generate the Artifact (and we skip publishing it later on)

Reply:
Fixed in last rework